Apollo.xcodeproj/project.pbxproj

```diff
@@ -3807,7 +3812,7 @@
 			repositoryURL = "https://github.com/daltoniam/Starscream.git";
 			requirement = {
 				kind = upToNextMinorVersion;
-				minimumVersion = 4.0.4;
+				minimumVersion = 3.1.1;
```

Comment:
It looks like there's some kind of reference to the main Starscream repo floating around in here